tests/e2e/module_file_manager/module_file_manager.cpp

```diff
@@ -0,0 +1,170 @@
+// Copyright 2022 Memgraph Ltd.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt; by using this file, you agree to be bound by the terms of the Business Source
+// License, and you may not use this file except in compliance with the Business Source License.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+#include <filesystem>
+#include <fstream>
+#include <ranges>
```

Comment:
:fire:

Reply:
You include it but you don't use it! <3

Reply:
I really tried but nothing warranted all the extra characters :smiling_face_with_tear: